src/app/components/practice/question-page/question-page.component.html

```diff
@@ -32,7 +32,7 @@
                 Problem Description
               </div>
               <div class="card-text">
-                {{question.body}}
+                <ng-katex-paragraph [paragraph]="body"></ng-katex-paragraph>
```

Comment:
Rather than refer to a new parameter called `body`, I think you can actually refer to `question.questionBody` here.

Reply:
Now you understand this.